### PR TITLE
Merge from Original

### PR DESCRIPTION
**Pull request by praneetmxit**
Praneet

### CODE REVIEW

File: application/views/partial/footer.php [hunk 1]
```diff
@@ -3,11 +3,6 @@
 
 	<div id="footer">
 		<div class="jumbotron push-spaces">
-			<strong><?php echo $this->lang->line('common_you_are_using_ospos'); ?>
```
Comment (daN4cat) [inline application/views/partial/footer.php]: You are breaching the License, you must reinstate the original footer and not change OSPOS with another name.
OR you cannot use this software.

File: application/views/login.php [hunk 2]
```diff
@@ -38,7 +38,7 @@
 			</div>
 		<?php echo form_close(); ?>
 		
-		<h1>Open Source Point Of Sale <?php echo $this->config->item('application_version'); ?></h1>
+		<h1>Colossus CXM <?php echo $this->config->item('application_version'); ?></h1>
```
Comment (daN4cat) [inline application/views/login.php]: You are not supposed to change the name, you are breaching the software license.
If you do that you cannot use the software.
This is the 2nd warning.

**Comment by jekkos** [comment]
Sad to see this happening again and again. We are people dedicating a lot of effort in this product and you are simply claiming this work to be your own. 

Please comply with the license or otherwise I will look how I can enforce it against you.

**Comment by praneetmxit** [comment]
We are going to add this to licencse to POS but we will be building on top our own suite of application.Regards,Praneet Chandra | (+91) 95-81-81-0789 | Colossus 360 | Socialize---- On Thu, 11 Aug 2016 16:55:52 +0530  jekkos<notifications@github.com> wrote ----Sad to see this happening again and again. We are people dedicating a lot of effort in this product and you are simply claiming this work to be your own.   Please comply with the license or otherwise I will look how I can enforce it against you.  —You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by jekkos** [comment]
https://help.github.com/articles/dmca-takedown-policy/

**Comment by jekkos** [comment]
@praneetmxit I'm afraid that if you do you'll need to keep the original footer.

**Comment by praneetmxit** [comment]
We will restore it with next commit.Regards,Praneet Chandra | (+91) 95-81-81-0789 | Colossus 360 | Socialize---- On Thu, 11 Aug 2016 17:07:46 +0530  jekkos<notifications@github.com> wrote ----@praneetmxit I'm afraid that if you do you'll need to keep the original footer.  —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by daN4cat** [comment]
@praneetmxit just to be clear, in general you are not allowed to rebrand any open source without copy righters consent. Also a license statement needs to be read and understood, and if something specific is mentioned you are required to comply.

**Pull request closed without merging**